client/data/hosting/use-site-logs-query.ts

```diff
@@ -5,12 +5,12 @@ interface SiteLogsAPIResponse {
 	message: string;
 	data: {
 		total_results: number;
-		logs: object[];
+		logs: Record< string, any >[];
```

Comment:
It's always a `string` or an `number` so maybe can use `Record< string, string | number >` here

Reply:
I wanted to protect against any future API changes in `renderCell()` by making sure we fell back back to something like `JSON.stringify` if we encountered something unexpected. We only get strings and numbers for now but it doesn't really document what the type is going to be.

But I probably should have gone with `unknown` instead of `any`. Since `unknown` will make sure we do checking before we use the value.